src/detect/i2cScan.h

```diff
@@ -114,10 +115,10 @@ void scanI2Cdevice(void)
                 screen_found = addr;
                 DEBUG_MSG("st7567 display found\n");
             }
-#ifdef HAS_AXP192
-            if (addr == AXP192_SLAVE_ADDRESS) {
+#if defined(HAS_AXP192) || defined(HAS_AXP2101)
+            if (addr == XPOWERS_AXP192_AXP2101_ADDRESS) {
                 axp192_found = true;
-                DEBUG_MSG("axp192 PMU found\n");
+                DEBUG_MSG("axp192/axp2101 PMU found\n");
```

Comment:
Both chips have the same I2C Address?

Reply:
AXP192 and AXP2101 have the same device address